js-intro.js

```diff
@@ -36,8 +62,17 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+removeVowel = (str) => {
+  let newArr1 = str.split("")
```

Comment:
Rather than `newArr` 1 & 2, consider naming your varibles with more purpose like `spliArr` and `filteredArr`. When your code starts to get really long, it's helpful to understand what you were trying to do by naming variables with intention.